Pocket_Calculator.c

```diff
@@ -0,0 +1,218 @@
+/* 
+ * File:   main.c
+ * Author: belmin
+ *
+ * Created on 28. September 2017, 17:32
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <float.h>
+#include <math.h>
+
+#define ADD 1
+#define SUBTRACT 2
+#define MULTIPLY 3
+#define DIVIDE 4
+#define STOP_PROGRAM -1
+#define POWER 5
+#define SQUARE_ROOT 6
+#define ARITHMETIC_SERIES 7
+
+#define TRUE 1
+#define FALSE 0
+
+double perform_operation(double, double, int);
+void get_operands(double*, double*, int);
+double multiply(double, double);
+double subtract(double, double);
+double add(double, double);
+int display_menu();
+// Extra Credit
+double perform_operation_with_just_one_operand(double, int);
+double arithmetic_series(double);
+double power(double, double);
+double square_root(double); // You have to add the option -lm to the call of the gcc command to build your program successfully.
+/*
+ * 
+ */
+int main() 
+{
+    double first_operand = 0.0;
+    double second_operand = 0.0;
+    double result = 0;    
+    
+    do
+    {
+        int choice = display_menu();

```

Comment:
PLEASE: never ever use while (TRUE) ... break! It is one of the ugliest styles of programming.